src/main/scala/in/ashwanthkumar/suuchi/partitioner/PartitionNode.scala

```diff
@@ -0,0 +1,6 @@
+package in.ashwanthkumar.suuchi.partitioner
+
+case class Node(id: Int, host: String)
```

Comment:
I thought about it too. If at all anything, We should use be using ```MemberAddress```. But, I will do it once I merge to master. ok?